src/main/scala/scalismo/mesh/MeshBoundaryPredicates.scala

```diff
@@ -77,11 +96,47 @@ private class BoundaryOfATriangleMeshPredicates(private var vertexIsOnBorder: In
   }
 }
 
+/**
+ * Implementation of a TetrahedralMeshBoundary
+ */
+private class BoundaryOfATetrahedralMeshPredicates(private var vertexIsOnBorder: IndexedSeq[Boolean],
```

Comment:
do these really need to be vars or would vals also do it?

Reply:
Changed to vals.